Kickstarter-iOS/Views/Controllers/FacebookConfirmationViewController.swift

```diff
@@ -60,10 +64,19 @@ internal final class FacebookConfirmationViewController: UIViewController,
     _ = self
       |> baseControllerStyle()
 
+    _ = self.disclaimerButton
+      |> disclaimerButtonStyle
+
     _ = self.confirmationLabel |> fbConfirmationMessageLabelStyle
     _ = self.createAccountButton |> createNewAccountButtonStyle
     _ = self.emailLabel |> fbConfirmEmailLabelStyle
-    _ = self.loginButton |> loginWithEmailButtonStyle
+
+    _ = self.loginButton
+      |> greyButtonStyle
+      |> UIButton.lens.title(for: .normal) %~ { _ in
+        Strings.login_buttons_log_in()
```

Comment:
If we are not using `loginWithEmailButtonStyle` anywhere else, I think you could change the string directly in there so the assignment is cleaner.